1581086, update image filenames part two 10-12 PM (#15145)
dotnet/docs: docs/framework/whats-new/whats-new-in-accessibility.md

To use this property, simply add it to your XAML code and use the appropriate brush or binding. The resulting text selection looks like this:

> ![Screenshot of what it looks like when you use the SelectionTextBrush.](./media/whats-new-in-accessibility/selectiontextbrush-property.png)

Reviewer: This alt text doesn't help someone who can't see the image.

It could be something like "Screenshot of the app running with the word Hello World selected."

A similar exercise should be done throughout the rest of this article.

  Before: 

>   ![Screenshot of the expander control with focus before improvements.](./media/whats-new-in-accessibility/expander-control-before.png)

Reviewer: ```suggestion
  ![Screenshot of the expander control with focus and no focus visual.](./media/whats-new-in-accessibility/expander-control-before.png)
```

  After: 

>   ![Screenshot of the expander control with focus after improvements.](./media/whats-new-in-accessibility/expander-control-after.png)

Reviewer: ```suggestion
  ![Screenshot of the expander control with focus showing a dotted line around the control's text.](./media/whats-new-in-accessibility/expander-control-after.png)
```

- <xref:System.Windows.Controls.CheckBox> and <xref:System.Windows.Controls.RadioButton> controls

  The text in the <xref:System.Windows.Controls.CheckBox> and <xref:System.Windows.Controls.RadioButton> controls is now easier to see when selected in high contrast themes. For example:

  Before: 

>   ![Screenshot of high contrast radio button before improvements.](./media/whats-new-in-accessibility/high-contrast-radio-button-before.png)

Reviewer: ```suggestion
  ![Screenshot of radio and check buttons with poor text visibility on high contrast themes.](./media/whats-new-in-accessibility/high-contrast-radio-button-before.png)
```

  After: 

>   ![Screenshot of high contrast radio button after improvements.](./media/whats-new-in-accessibility/high-contrast-radio-button-after.png)

Reviewer: ```suggestion
  ![Screenshot of radio and check buttons with better text visibility on high contrast themes.](./media/whats-new-in-accessibility/high-contrast-radio-button-after.png)
```

- <xref:System.Windows.Controls.ComboBox> control

  Starting with .NET Framework 4.7.1, the border of a disabled <xref:System.Windows.Controls.ComboBox> control is the same color as disabled text. For example:

  Before: 

>   ![Screenshot of ComboBox disabled before accessibility improvements.](./media/whats-new-in-accessibility/combo-disabled-before.png)

Reviewer: ```suggestion
  ![Screenshot of a disabled ComboBox with border and control text in different colors.](./media/whats-new-in-accessibility/combo-disabled-before.png)
```

  After:   

>   ![Screenshot of ComboBox disabled after accessibility improvements.](./media/whats-new-in-accessibility/combo-disabled-after.png)

Reviewer: ```suggestion
  ![Screenshot of a disabled ComboBox with border the same color as the control text.](./media/whats-new-in-accessibility/combo-disabled-after.png)
```

  In addition, disabled and focused buttons use the correct theme color.

  Before:

>   ![Screenshot of button theme colors before accessibility improvements.](./media/whats-new-in-accessibility/button-theme-colors-before.png) 

Reviewer: ```suggestion
  ![Screenshot of a black button with gray text saying Focus Me.](./media/whats-new-in-accessibility/button-theme-colors-before.png) 
```

  After: 

>   ![Screenshot of button theme colors after accessibility improvements.](./media/whats-new-in-accessibility/button-theme-colors-after.png) 

Reviewer: ```suggestion
  ![Screenshot of a blue button with black text saying Focus Me.](./media/whats-new-in-accessibility/button-theme-colors-after.png) 
```

  Finally, in .NET Framework 4.7 and earlier versions, setting a <xref:System.Windows.Controls.ComboBox> control’s style to `Toolbar.ComboBoxStyleKey` caused the drop-down arrow to be invisible. This issue is fixed starting with .NET Framework 4.7.1. For example:

  Before: 

>   ![Screenshot of Toolbar.ComboBoxStyleKey before improvements.](./media/whats-new-in-accessibility/combo-box-style-key-before.png) 

Reviewer: ```suggestion
  ![Screenshot of a ComboBox control with an invisible drop-down arrow.](./media/whats-new-in-accessibility/combo-box-style-key-before.png) 
```

  After: 

>   ![Screenshot of Toolbar.ComboBoxStyleKey after improvements.](./media/whats-new-in-accessibility/combo-box-style-key-after.png) 

Reviewer: ```suggestion
  ![Screenshot of a ComBoxBox control displaying the drop-down arrow.](./media/whats-new-in-accessibility/combo-box-style-key-after.png) 
```

- <xref:System.Windows.Controls.DataGrid> control

  Starting with .NET Framework 4.7.1, the sort indicator arrow in <xref:System.Windows.Controls.DataGrid> controls now uses correct theme colors. For example:

  Before: 

>   ![Screenshot of sort indicator arrow before improvements.](./media/whats-new-in-accessibility/sort-indicator-before.png) 

Reviewer: These images actually look wrong. I don't see much difference between them and I'm not seeing a sort indicator arrow.

@merriemcgaw am I missing something?
![image](https://user-images.githubusercontent.com/12971179/69317914-d57cbd80-0bf0-11ea-8cd7-38f86499b174.png)

Author: @mairaw no, I don't think you're missing anything. I didn't notice this when I was working on the article, but you are correct. I checked the published article and I'm seeing the same anomaly: [What's new in accessibility in .NET Framework 4.7.1](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility#whats-new-in-accessibility-in-net-framework-471).

Is there any action for me to take, or is this something your team will fix? Happy to help in any way I can.

Reviewer: I'll file an issue for this.

  After:   

  ![Screenshot of sort indicator arrow after improvements.](./media/whats-new-in-accessibility/sort-indicator-after.png) 

  In addition, in .NET Framework 4.7 and earlier versions, the default link style changed to an incorrect color on mouse over in high contrast modes. This is resolved starting with .NET Framework 4.7.1. Similarly, <xref:System.Windows.Controls.DataGrid> checkbox columns uses the expected colors for keyboard focus feedback starting with .NET Framework 4.7.1.

  Before: 

>   ![Screenshot of DataGrid default link style before improvements.](./media/whats-new-in-accessibility/default-link-style-before.png) 

Reviewer: ```suggestion
  ![Screenshot of a link saying Click Me! in red.](./media/whats-new-in-accessibility/default-link-style-before.png) 
```

  After:    

>   ![Screenshot of DataGrid default link style after improvements.](./media/whats-new-in-accessibility/default-link-style-after.png) 

Reviewer: ```suggestion
  ![Screenshot of a link saying Click Me! in yellow.](./media/whats-new-in-accessibility/default-link-style-after.png) 
```

  Before:

>   ![Screenshot of disabled text before accessibility improvements.](./media/whats-new-in-accessibility/wf-disabled-menu-item-before.png) 

Reviewer: ```suggestion
  ![Screenshot of an app that uses different controls running in high contrast mode before accessibility improvements.](./media/whats-new-in-accessibility/wf-disabled-menu-item-before.png) 
```

Reviewer: You could call this image something to indicate high-contrast-mode-before and then the other one after.

  After:

>   ![Screenshot of disabled text after accessibility improvements.](./media/whats-new-in-accessibility/wf-disabled-menu-item-after.png) 

Reviewer: ```suggestion
  ![Screenshot of an app that uses different controls running in high contrast mode after accessibility improvements.](./media/whats-new-in-accessibility/wf-disabled-menu-item-after.png) 
```
